pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -282,6 +293,121 @@ func configMapDeleted(obj interface{}) {
 	}
 }
 
+// initVolumeHandleToPvcMap performs all the operations required to initialize the volume id to PVC name map.
+// It also watches for PV update & delete operations, and updates the map accordingly.
+func initVolumeHandleToPvcMap(ctx context.Context, k8sClient clientset.Interface) {
+	log := logger.GetLogger(ctx)
+	log.Debugf("Initializing volume ID to PVC name map")
+	k8sOrchestratorInstance.volumeIDToPvcMap = make(map[string]string)
+
+	// Set up kubernetes resource listener to listen events on PersistentVolume
+	k8sOrchestratorInstance.informerManager = k8s.NewInformer(k8sClient)
```

Comment:
We are setting the `informerManager` here as well - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go#L197. Can we make sure the `informerManager` is set in one place?